src/sys/windows.rs

```diff
@@ -882,6 +891,12 @@ impl Socket {
     }
 }
 
+impl Drop for Socket {
```

Comment:
Revert this; now `crate::Socket` is closed twice.

Reply:
Same as above.